README.md

```diff
@@ -1 +1,91 @@
-# WHO-Covid19
\ No newline at end of file
+# WHO Covid19
+
+>WHO Covid19 is a web application for presenting numerical data regarding the covid-19 pandemic in South America. The used API provides statistics via REST API from The World Health Organization Situation Reports, Johns Hopkins CSSE, The U.S. Department of Health & Human Services, The National Health Commission of the People’s Republic of China, The European Centre for Disease Prevention and Control, and China CDC Weekly. This API retrieves data by country, including population, the number of the virus confirmed, recovered, critical cases, deaths, recovered per death ratio, cases per million population, and more. The data is updated multiple times a day.
+
+
+## Built With
+
+- HTML
+- JavaScript
+- CSS
+- Git
+- React
+- Redux
+
+
+## Live Demo (if available)
+
+[Project Video](https://www.loom.com/share/5191b4804ec64ff9a87d1a4f6e43bfb9)
```

Comment:
- [ ] Kindly be noticed that the video should contain a highlight about a piece of the code you finds interesting or proud of making.